main.js

```diff
@@ -0,0 +1,35 @@
+let myName = "Vladyslav";
+
+let mySurmame = "Klokun";
+
+let myCity = "Kyiv";
+
+let average = 10 + 12 + 10 + 5 / 4;
```

Comment:
```suggestion
let average = (10 + 12 + 10 + 5)  / 4;
```
Don't forget about arithmetic operators' priority, the result of the operation is wrong.